src/Task/TwigCsFixer.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GrumPHP\Task;
+
+use GrumPHP\Formatter\ProcessFormatterInterface;
+use GrumPHP\Runner\TaskResult;
+use GrumPHP\Runner\TaskResultInterface;
+use GrumPHP\Task\Config\ConfigOptionsResolver;
+use GrumPHP\Task\Context\ContextInterface;
+use GrumPHP\Task\Context\GitPreCommitContext;
+use GrumPHP\Task\Context\RunContext;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use GrumPHP\Fixer\Provider\FixableProcessResultProvider;
+use Symfony\Component\Process\Process;
+
+/**
+ * @extends AbstractExternalTask<ProcessFormatterInterface>
+ */
+class TwigCsFixer extends AbstractExternalTask
+{
+    public static function getConfigurableOptions(): ConfigOptionsResolver
+    {
+        $resolver = new OptionsResolver();
+        $resolver->setDefaults([
+            'paths' => ['.'],
+            'level' => 'NOTICE',
+            'config' => null,
+            'report' => 'text',
+            'fix' => false,
+            'no-cache' => false,
+            'debug' => false,
+            'quiet' => false,
+            'version' => false,
+            'ansi' => false,
```

Comment:
Tasks are being ran in sub-processes.
Having ansi / no-ansi configurable does not make a lot of sense for grumphp.

The same goes for no-interaction : it will have no interaction by default since it is being ran in a subprocess in parallel.

Reply:
Done!